examples/dft/compile_time_dispatching/complex_fwd_buffer_mklgpu.cpp

```diff
@@ -43,80 +28,76 @@
 #endif
 #include "oneapi/mkl.hpp"
 
-// local includes
-#include "example_helper.hpp"
-
-void run_getrs_example(const sycl::device& cpu_device) {
-    // Matrix sizes and leading dimensions
-    constexpr std::size_t N = 10;
-    std::int64_t rs[3] {0, N, 1};
+void run_example(const sycl::device& gpu_deviceice) {
```

Comment:
`gpu_deviceice` -> `gpu_device`

Reply:
Done in [95f4e2c](https://github.com/hjabird/oneMKL/pull/6/commits/95f4e2ce326ac7a1c78068056ff9bbf3a400d705).